lib/CMakeLists.txt

```diff
@@ -54,6 +54,12 @@ set (LIBS
     ${LEATHERMAN_LIBRARIES}
 )
 
+if (WIN32)
+    # Necessary when statically linking cpp-pcp-client on Windows.
+    # Shouldn't hurt when cpp-pcp-client is a DLL.
```

Comment:
Can't we just test what kind of cpp-pcp-client library we are linking against? Whether static or shared?